OmegaHoverNerf.mlua

```diff
@@ -0,0 +1,116 @@
+--[Patch]--
+Title("Omega Hover Nerf")
+Author("Rei-San(Reimous,GhostTH)")
+Platform("Xbox 360")
+Blurb("Limits Omega Hovering")
```

Comment:
Change the Blurb field to `Limits Omega's hovering to only a few seconds to prevent exploits.`